ontology/uco/observable/observable.ttl

```diff
@@ -6069,6 +6330,46 @@ observable:Server
 	sh:targetClass observable:Server ;
 	.
 
+observable:Service
+	a
+		owl:Class ,
+		sh:NodeShape
+		;
+	rdfs:subClassOf observable:Process ;
+	.
+
+observable:ServicePack
+	a
+		owl:Class ,
+		sh:NodeShape
+		;
+	rdfs:subClassOf observable:Software ;
+	.
+
+observable:SetConfigurationEntry
```

Comment:
@sbarnum - Could you please confirm the subclass hierarchy under `SetConfigurationEntry` is as you intended?